package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "next lint --dir ./src ./tests",
```

Comment:
we're adding an ./amplify directory imminently that we'll also want to lint - is it possible just to exclude folders we don't want to lint so that we aren't accidentally adding new stuff that we aren't linting?

Reply:
We're using next lint (perhaps we should move to eslint directly) which by default only checks `pages`, `app` `components`, 'lib' and `src`.

Reply:
I've updated the lint command to be `next lint --dir .` and added a `.eslintignore` and exlcuded `./public` which contains all our assets and the `jest.*.ts` files which complain about wanting dev deps as not dev deps.